config/messaging.kafka.yml

```diff
@@ -1,9 +1,12 @@
 ---
 base: &base
-  hostname: localhost
+  host: localhost
```

Comment:
Does this require a change in appliance_console?  Part of me is thinking keep this as hostname so it's consistent with database.yml.

Reply:
Yes, linked in the description https://github.com/ManageIQ/manageiq-appliance_console/pull/197

Reply:
I'm okay keeping it consistent with database.yml, I'm thinking we should update manageiq-messaging though instead to take `:hostname` so that we don't have to update the options in a few of different places?  At a minimum we have the code in MiqQueue and manageiq-messaging-ready which can't load the env to reuse this code.